golang/cosmos/third_party/proto/google/protobuf/any.proto

```diff
@@ -35,7 +35,7 @@ package google.protobuf;
 import "gogoproto/gogo.proto";
 
 option csharp_namespace = "Google.Protobuf.WellKnownTypes";
-option go_package = "types";
+option go_package = "github.com/cosmos/cosmos-sdk/codec/types";
```

Comment:
I don't think this affects any of our code (given the explicit remap of `google/protobuf/any.proto` in the `protogen.sh`), but I would prefer not to diverge unnecessarily from `$GOPATH/include/google/protobuf/any.proto`.  I think this should be:

```suggestion
option go_package = "github.com/golang/protobuf/ptypes/any";
```

Reply:
Done.